common/install.sh

```diff
@@ -24,6 +24,11 @@ mkdir -p $WELLBEING_PREF_FOLDER
 cp -p $WELLBEING_PREF_FILE $WELLBEING_PREF_FOLDER
 am force-stop "com.google.android.apps.wellbeing"
 
+ui_print " "
+ui_print "   Fixing Permissions for Sounds V2.0..."
```

Comment:
Should we really say ``Sounds V2.0`` as opposed to just ``Fixing permissions for Sounds...``? (``permissions`` should be lowercase)